docs/DeveloperGuide.md

```diff
@@ -365,20 +365,24 @@ Refer to the guide [here](./DevOps.md)
 
 **Target user profile**:
 
+* university students
 * take multiple modules
-* want to easily track their weekly calendar
-* want to be notified about important academic dates coming up
-* want to have their personal todos & events integrated with their academic calendar
+* want to easily know the events lined up in their week ahead
+* prefer to have everything in one application
```

Comment:
I think "to have everything" sounds "broad" and "big", maybe we should make it more specific or more reader-friendly?